lib/beaker/host/pswindows/file.rb

```diff
@@ -2,17 +2,29 @@ module PSWindows::File
   include Beaker::CommandFactory
 
   def tmpfile(name)
-    execute("echo C:\\Windows\\Temp\\#{name}.%RANDOM%")
+    execute("echo #{system_temp_path}\\#{name}.%RANDOM%")
   end
 
   def tmpdir(name)
     # generate name
-    tmpdirname = execute("echo C:\\Windows\\Temp\\#{name}.%RANDOM%")
-    # created named dir
+    tmpdirname = tmpfile(name)
+    # create named dir
     execute("md #{tmpdirname}")
     tmpdirname
   end
 
+  # Turns a host file path into one that can be used by BitVise SSH
+  #
+  # @param [String] path Path to be changed
+  #
+  # @return [String] network version of that path
+  def network_path(path)
```

Comment:
Also, I see where you're going with `network_path`, but it's a pretty generic term - I think `scp_path` might be more appropriate.

That said, this still seems like the wrong place to do this.  Might it be better to normalize paths inside the SCP helpers instead?  That way there aren't calls to `network_path` all over the code base every time such a request is initiated.  Let the slashification be deferred until copy time, keeping in mind that for Bitvise, `C:\` should become `/`?

/cc @joshcooper @ferventcoder